packages/emoji-mart/src/components/Picker/Picker.tsx

```diff
@@ -1069,7 +1069,14 @@ export default class Picker extends Component {
 
   renderLiveRegion() {
     const emoji = this.getEmojiByPos(this.state.pos)
-    const contents = emoji ? emoji.name : ''
+    const noSearchResults =
+      this.state.searchResults && !this.state.searchResults.length
```

Comment:
This condition doesn't seem to be entirely correct since what if `this.state.searchResults` is undefined?

Reply:
It's the same condition used to render the preview (where the `no result` component is rendered) https://github.com/slidoapp/emoji-mart/blob/c39c09970fa816977a535be4caf12d9dd29f91e4/packages/emoji-mart/src/components/Picker/Picker.tsx#L718, I just copied it from there.

Reply:
Word of caution, we took over this repo but the code quality is as before, which means not ideal, so I wouldn't trust the code completely. Either `this.state.searchResults` is never falsy and you don't need the check:
```suggestion
    const noSearchResults = !this.state.searchResults.length
```

or it can actually be falsy and then it can happen `noSearchResults` will be `falsy` even though there won't be no search results, do you see the dilemma? So I'd rather be explicit about it:

```suggestion
    const noSearchResults =
      this.state.searchResults == null || this.state.searchResults.length === 0
```

Reply:
Gotcha. I've used your suggestion.